<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список выдачей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Название книги</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_BookName"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Дата выдачи</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineDateOut"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Дата возврата</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineDateIN">
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnReset">
           <property name="text">
            <string>Очистить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Список выдачей</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableWidget" name="table_book_out">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>id книги</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Дата выдачи</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Дата возврата</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
